data/resume/jbatorski.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jsonresume/resume-schema/v1.0.0/schema.json",
+  "basics": {
+    "name": "Jerry Batorski",
+    "label": "Frontend Software Engineer",
+    "email": "jerry.batorski@gmail.com",
+    "url": "https://jarekb84.github.io/me/",
+    "summaryOfQualifications": [
+      "11+ YOE as a software engineer, with a focus on the frontend",
+      "9+ YOE breaking down projects, communicating with stakeholders, delivering business value, and adapting to change",
+      "3+ YOE leading a frontend focused squad of software engineers"
+    ]
+  },
+  "work": [
+    {
+      "name": "Grubhub",
+      "location": "Chicago, IL",
+      "url": "https://www.grubhub.com/",
+      "startDate": "2016-11-01",
```

Comment:
Lack of the end date property is an indicator that this is the current job I'm at. I considered adding it with a value of `Present` or `Current`, but the [jasonresume schema expects an ISO8601](https://github.com/jsonresume/resume-schema/blob/master/schema.json#L132) date format here. So will need to check for this property being missing on the react side and display some text instead.